iphone/Classes/TiUIListView.m

```diff
@@ -825,6 +825,14 @@ -(void)setDisableBounce_:(id)value
 {
     [[self tableView] setBounces:![TiUtils boolValue:value def:NO]];
 }
+-(void)setAllowsMultipleSelectionDuringEditing_:(id)args
```

Comment:
Use:
```objc
- (void)setAllowsMultipleSelectionDuringEditing_:(id)value
 {
    ENSURE_TYPE(value, NSNumber);

    [[self tableView] beginUpdates];
    [[self tableView] setAllowsMultipleSelectionDuringEditing:[TiUtils boolValue:value]];
    [[self tableView] endUpdates];

    [[self proxy] replaceValue:value forKey:@"allowsMultipleSelectionDuringEditing_" notification:NO];
 }
```
+ are you sure you need the `beginUpdates` wrapper here? It's only required for some API's.